fix: Fixed autobuild on windows

munsterlander opened the pull request:
# Description
<!--
Provide a description of what this PR is doing.
If you're modifying existing behavior, describe the existing behavior, how this PR is changing it,
and what motivated the change. If this is a breaking change, specify explicitly which APIs were
changed.
-->
There are issues for live doc build in Sphinx on windows.  The batch file doesn't properly change commands.  This fixes that based on conversations in: https://github.com/flame-engine/flame/issues/2075, https://github.com/flame-engine/flame/pull/2082, and discord.

I will need to update the documentation to add that if you are running autobuild on Windows you need to run:

`pip install sphinx-autobuild` as it is not installed by default.

## Checklist
<!--
Before you create this PR confirm that it meets all requirements listed below by checking the
relevant checkboxes with `[x]`. If some checkbox is not applicable, mark it as `[-]`.
-->

- [x] The title of my PR starts with a [Conventional Commit] prefix (`fix:`, `feat:`, `docs:` etc).
- [x] I have followed the [Contributor Guide] when preparing my PR.
- [ ] I have updated/added tests for ALL new/updated/fixed functionality.
- [ ] I have updated/added relevant documentation in `docs` and added dartdoc comments with `///`.
- [ ] I have updated/added relevant examples in `examples` or `docs`.


## Breaking Change?
<!--
Would your PR require Flame users to update their apps following your change?

If yes, then the title of the PR should include "!" (for example, `feat!:`, `fix!:`). See
[Conventional Commit] for details. Also, for a breaking PR uncomment and fill in the "Migration
instructions" section below.

### Migration instructions

If the PR is breaking, uncomment this header and add instructions for how to migrate from the
currently released version to the new proposed way.
-->

- [ ] Yes, this PR is a breaking change.
- [x] No, this PR is not a breaking change.


## Related Issues
<!--
Indicate which issues this PR resolves, if any. For example:
Closes #1234
!-->

<!-- Links -->
[Contributor Guide]: https://github.com/flame-engine/flame/blob/main/CONTRIBUTING.md
[Conventional Commit]: https://conventionalcommits.org

spydon commented:
> pip install sphinx-autobuild as it is not installed by default.

It is installed if the docs are followed, check in the `requirements.txt` file.

spydon commented on doc/_sphinx/make.bat:
```diff
@@ -26,11 +27,16 @@ if errorlevel 9009 (
 )
 
-%SPHINXBUILD% -M %1 %SOURCEDIR% %BUILDDIR% %SPHINXOPTS% %O%
+if "%1" == "livehtml" (
```
Is this really how one does make targets on windows?
I would have guessed it is more like with `:help` in the end of the file.

spydon replied:
Maybe there is no clean way of doing it, but this way feels more like a makefile:
https://superuser.com/a/717465

munsterlander replied:
Well you can do it several ways.  If statements or named blocks.

munsterlander replied:
Updated for blocks.  Its the same difference.

munsterlander commented:
> > pip install sphinx-autobuild as it is not installed by default.
> 
> It is installed if the docs are followed, check in the `requirements.txt` file.

Well I did and it didn't on windows, lol.  It could have been several issues, but we addressed this in the discord thread and changed docs to simply say, verify that all modules installed correctly.  

I would add that even in the make.bat file, there is output about setting either a path entry or creating an environment variable, so I suspect there could be other environmental issues beyond what is expected, thus the verify everything installed correctly is probably the best approach.
